src/ui/components/Shared/TracksListHeaderCell.react.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Icon from 'react-fontawesome';
+
+import LibraryActions from '../../actions/LibraryActions';
+
+
+class TracksListHeaderCell extends React.Component {
+  static propTypes = {
+    className: PropTypes.string,
+    sortBy: PropTypes.string,
+    title: PropTypes.string,
+    onClick: PropTypes.func,
+    icon: PropTypes.string,
+  }
+
+  constructor(props) {
+    super(props);
+    this.sort = this.sort.bind(this);
+  }
+
+  sort() {
+    LibraryActions.sort(this.props.sortBy);
+  }
+
+  getCellContent(props) {
+    return (
+      <React.Fragment>
+        <div className='col-name'>
+          {props.title}
+        </div>
+        {props.icon &&
+          <div className='col-icon'>
+            <Icon name={props.icon} />
+          </div>
+        }
```

Comment:
why not just use plain `if`?

Reply:
How that?

Reply:
I thought you could use plain if there, nevermind